.github/workflows/lint.yml

```diff
@@ -0,0 +1,8 @@
+name: Ruff lint
+on: pull_request
```

Comment:
why not `on: [push, pull_request]` similar to other workflows? :thinking:

Reply:
right, thought was too much for every push. But since this is a small workflow it does not hurt. Will apply also to push